src/dot.cpp

```diff
@@ -396,7 +395,17 @@ static bool convertMapFile(FTextStream &t,const char *mapName,
 
       if (buf.left(5)=="<area")
       {
-        t << replaceRef(buf,relPath,urlOnly,context).replace(re,"");
+	QCString replBuf = replaceRef(buf,relPath,urlOnly,context);
+        int indexS = replBuf.find("id=\""), indexE;
+        indexE=replBuf.find('"',indexS+4);
+        if (indexS>=0 && (indexE=buf.find('"',indexS))!=-1)
```

Comment:
This doesn’t look right. `indexE` is already assigned, you probably meant `indexE>indexS`?

Reply:
Looks indeed like some initial debug statement remained, I'll have a look (results were OK but this code isn't).

Reply:
Corrected code.